core/templates/dev/head/pages/exploration_editor/translation_tab/VoiceArtistOverviewDirective.js

```diff
@@ -19,7 +19,7 @@
 
 oppia.constant('DEFAULT_AUDIO_LANGUAGE', 'en');
 
-oppia.directive('translatorOverview', [
+oppia.directive('voiceArtistOverview', [
```

Comment:
Undo these changes, our current translation tab use to handle both translation and voiceover we have to think about the name of the tab and do the changes as required. (see #5854)

Reply:
Done. I changed everything related to translatorOverview back, if that is how it is supposed to be.